docs/en/observability/view-observability-alerts.asciidoc

```diff
@@ -0,0 +1,108 @@
+
+****
+Given the experimental status of this feature, if you need to disable and remove the Alerts page,
+you can do so using the following configuration settings:
+
+`xpack.ruleRegistry.write.enabled: 'false'`
+`xpack.observability.unsafe.cases.enabled: 'false'`
+`xpack.observability.unsafe.alertingExperience.enabled: 'false'`
+
+If you have a self-managed {stack}, edit the settings in your `kibana.yml` file.
+
+If you are using our https://www.elastic.co/cloud/elasticsearch-service[hosted {ess}] on {ecloud},
+edit the {kib} user settings:
+
+. Select your deployment on the home page, and from your deployment menu,
+go to the *Edit* page.
+. In the *{kib}* section, click *Edit user settings*, and add the settings if they are not already present.
+. Click *Back*, and then click *Save*. The changes are
+automatically appended to the `kibana.yml` configuration file for your instance.
+
+****
+
+The Alerts page lists all the alerts that have met a condition defined by a rule you created using the Logs,
+Metrics, Uptime, or APM apps.
+
+After alerts have been triggered, you can monitor their activity to verify they are functioning correctly.
+In addition, you can filter alerts, update their workflow status, and troubleshoot each alert in their
+respective app.
+
+You can also add alerts to <<create-cases,Cases>> to open and track potential infrastructure issues.
+
+NOTE: You can centrally manage rules from the
+{kibana-ref}/create-and-manage-rules.html[{kib} Management UI] that provides a
+set of built-in {kibana-ref}/rule-types.html[rule types] and
+{kibana-ref}/action-types.html[connectors] for you to use. Click *Manage Rules*
+
+[role="screenshot"]
+image::images/alerts-page.png[Alerts page]
+
+[discrete]
+[[filter-observability-alerts]]
+== Filter alerts
+
+To help you get started with your analysis faster, use the KQL bar to create structured queries using
+{kibana-ref}/kuery-query.html[Kibana Query Language]. For example, `kibana.alert.status: "recovered"`.
+
+You can also use the time filter to define a specific date and time range. By default, this filter is set to search
+for the last 15 minutes.
+
+To view alerts that you have already taken action on, select either the *Acknowledged* or *Closed* tabs.
+
+[discrete]
+[[view--alert-details]]
+== View alert details
+
+When you have searched and filtered for a specific alert, you may want to inspect the alert details.
+From the Alerts table, select image:images/flyout-icon.png[View details] to view additional information summarising the alert.
+
+You can find the current status of the alert, along with its duration, and when it was last
+updated. To help you determine what caused the alert, you can view the expected and actual
+threshold values, and the rule that produced the alert.
+
+To view the alert in the specific app it was initially created from, select image:images/app-link-icon.png[Link to app icon]. 
+
+[discrete]
+[[action-observability-alerts]]
+== Take action on alerts
```

Comment:
@EamonnTP This will soon be removed https://github.com/elastic/kibana/issues/117686. Users won't have the option to interact with the workflow status anymore. Does this documentation refer to 7.16?

Reply:
Hi @mgiota Yes, these updates are for 7.16. I created this [issue](https://github.com/elastic/observability-docs/issues/1260) to track the changes for 8.0.